nodejs/week1/warmup-mealcalculator/src/backend/routes/large_meals.js

```diff
@@ -0,0 +1,24 @@
+// Respond with the json for all the meals (including it's reviews) that can fit lots of people
+
+
+const mealsArray = require("../data/meals.json");
+const reviewsArray = require("../data/reviews.json");
+
+
+const getLargemeals = function(request, response){
+    
+    const mealreviewsArray = mealsArray.map(mealitem=>{
+        if(mealitem.maxNumberOfGuests >= 10){
+            mealitem.reviews = reviewsArray.filter(element => mealitem.id === element.mealId)
+               return mealitem;
+            }else{
+                return
```

Comment:
You're returning undefined here implicitly. It's better to use filter to remove the elements which you don't want to have in result instead 😉